tests/test_app.py

```diff
@@ -234,7 +234,18 @@ def test_setup_logging_with_envvar():
             encoding=ENCODING,
         )
         logging.getLogger.assert_called_once_with()
-        # ~ assert sys.excepthook == excepthook
+        # clear this to avoid interfering with other tests
+        os.environ.pop("MU_LOG_TO_STDOUT", "")
+
+
+def test_setup_except_hook():
+    """
+    confirm that setup_exception_handler() is setting up the global exception hook
+    """
+    saved = sys.excepthook
+    setup_exception_handler()
+    assert sys.excepthook == excepthook
+    sys.excepthook = saved
 
```

Comment:
Since we now setup the exception handler always (not just if logging is turned on) and so this test just confirms that it's set up by `setup_exception_handler()`